packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -480,7 +480,8 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         bytes memory ancillaryData,
         bytes32 hash
     ) public override nonReentrant() onlyIfNotMigrated() {
-        uint256 currentRoundId = getCurrentRoundId();
+        uint32 currentRoundId = SafeCast.toUint32(getCurrentRoundId());
+        (getCurrentRoundId());
```

Comment:
What does this line do?

Reply:
`currentRoundId` is `uint32`. I cast it at the top so that assignment would be ok with it later down the line:
<img width="810" alt="2022-08-26 at 11 23 24@2x" src="https://user-images.githubusercontent.com/12886084/186872780-331e4bb2-f1cd-43ac-bded-eaf203974b84.png">

I can also just unsafe cast it at the assignment point

Reply:
Sorry I wasn't asking about the cast, I was asking about line 484.

Reply:
you right, this is wrong. removed